src/sknnr/transformers/__init__.py

```diff
@@ -0,0 +1,3 @@
+from ._standard_scaler_dof import StandardScalerWithDOF
+
+__all__ = ["StandardScalerWithDOF"]
```

Comment:
Fully agree.  I've gone ahead and done that here (feature creep!).  Check my logic, but we probably don't need a new set of tests for transformers to make sure they implement `fit`, `transform`, and `fit_transform`, because they are encapsulated within the estimators and `fit` and `transform` have to be working in order for the estimators to pass.  I was looking at coverage and noted that we aren't necessarily testing `fit_transform` on any of these.

Reply:
> Check my logic, but we probably don't need a new set of tests for transformers to make sure they implement fit, transform, and fit_transform, because they are encapsulated within the estimators and fit and transform have to be working in order for the estimators to pass.

Yeah, agreed! I think the general advice is to test higher-level integrations over lower-level details, so testing the transformers via the estimators makes sense to me.

> I was looking at coverage and noted that we aren't necessarily testing fit_transform on any of these.

Good point. It looks like [fit_transform](https://github.com/scikit-learn/scikit-learn/blob/364c77e047ca08a95862becf40a04fe9d4cd2c98/sklearn/base.py#L876-L881) literally just calls `fit` then `transform`, so I don't think we're missing anything with that coverage.